cmd/deploy/remote.go

```diff
@@ -62,6 +64,10 @@ ARG {{ .TokenArgName }}
 ARG {{ .ActionNameArgName }}
 ARG {{ .TlsCertBase64ArgName }}
 ARG {{ .InternalServerName }}
+ARG {{ .OktetoDeployable }}
+ARG {{ .GitHubRepositoryArgName }}
+ARG {{ .BuildKitHostArgName }}
+ARG {{ .OktetoRegistryURLArgName }}
```

Comment:
These were added because in the previous way, the local execution of the command within BuildKit these were available there for some internal logic we have. Now, the runner doesn't execute that logic, so this would need to be in the environment to prevent a breaking change on someone relying on these env vars